### ISSUE TITLE
Sentry is not a constructor

### ISSUE THREAD

**Issue by notflip** [franciscofsales/winston-sentry-log#11]
When calling new Sentry() I get the error, Sentry is not a constructor

```
const Sentry = require('winston-sentry-log');
console.log(new Sentry({}));
```

**Comment by dottodot** [comment]
Same here.

**Comment by dogenius01** [comment]
same issue.

**Comment by franciscofsales** [comment]
fixed on #12 , verion 1.0.10 available on npm
